data/schema/core/actionwml.cfg

```diff
@@ -369,6 +369,8 @@
 		{LINK_TAG "units/$modifications/advancement"}
 		{LINK_TAG "units/$modifications/object"}
 		{LINK_TAG "units/$modifications/base/effect"}
+        {LINK_TAG "$action_wml/clear_variable"}
```

Comment:
Some weird spacing here?

Reply:
No idea how the tabulator got in there, fixed with the latest commit.

Will add your dialogue adjustments later and rebase accordingly.

Reply:
Hm, actually, should I rebase this? Since it shows you as co-author, how I have no idea how that would turn out.

Reply:
It's better to rebase it yes.

Reply:
Done.